patches/@supercolony+typechain-polkadot+0.5.0.patch

```diff
@@ -0,0 +1,211 @@
+diff --git a/node_modules/@supercolony/typechain-polkadot/.DS_Store b/node_modules/@supercolony/typechain-polkadot/.DS_Store
```

Comment:
What is this new patch solving ?

Reply:
This part has nothing to do with my intention; just slipped into it. Will fix it. For the other patches (following diffs) are to handle weight v2.